apps/nextra/public/robots.txt

```diff
@@ -0,0 +1,10 @@
+# *
```

Comment:
i'm guessing this stuff that was generated should be taken out??

Reply:
Yeah